sdk/zksync.js/src/provider.ts

```diff
@@ -44,35 +44,20 @@ export async function getDefaultProvider(
         } else if (transport === 'HTTP') {
             return await Provider.newHttpProvider('http://127.0.0.1:3030', pollIntervalMilliSecs);
         }
-    } else if (network === 'ropsten') {
+    } else if (network === 'testnet') {
         if (transport === 'WS') {
-            return await Provider.newWebsocketProvider('wss://ropsten-api.zksync.io/jsrpc-ws');
+            return await Provider.newWebsocketProvider('wss://dev.aggregation.rifcomputing.net:3031');
         } else if (transport === 'HTTP') {
-            return await Provider.newHttpProvider('https://ropsten-api.zksync.io/jsrpc', pollIntervalMilliSecs);
-        }
-    } else if (network === 'rinkeby') {
-        if (transport === 'WS') {
-            return await Provider.newWebsocketProvider('wss://rinkeby-api.zksync.io/jsrpc-ws');
-        } else if (transport === 'HTTP') {
-            return await Provider.newHttpProvider('https://rinkeby-api.zksync.io/jsrpc', pollIntervalMilliSecs);
-        }
-    } else if (network === 'ropsten-beta') {
-        if (transport === 'WS') {
-            return await Provider.newWebsocketProvider('wss://ropsten-beta-api.zksync.io/jsrpc-ws');
-        } else if (transport === 'HTTP') {
-            return await Provider.newHttpProvider('https://ropsten-beta-api.zksync.io/jsrpc', pollIntervalMilliSecs);
-        }
-    } else if (network === 'rinkeby-beta') {
-        if (transport === 'WS') {
-            return await Provider.newWebsocketProvider('wss://rinkeby-beta-api.zksync.io/jsrpc-ws');
-        } else if (transport === 'HTTP') {
-            return await Provider.newHttpProvider('https://rinkeby-beta-api.zksync.io/jsrpc', pollIntervalMilliSecs);
```

Comment:
thought we wanted to keep the ethereum configs for compatibility @antomor ?

Reply:
I decide to remove it to have a clean setup. If we ever need that reference, they are still on the git history. But please let me know if I should restore that.

Reply:
Yes, i think we should keep the ethereum configs to allow the users to use our SDK with any network eventually.